test/abi/Inputs/macOS/arm64/observation/baseline

```diff
@@ -31,13 +31,18 @@ _$s11Observation0A9RegistrarVSHAAMc
 _$s11Observation0A9RegistrarVSQAAMc
 _$s11Observation0A9RegistrarVSeAAMc
 _$s11Observation10ObservableMp
+_$s11Observation8ObservedV13untilFinishedyACyxq_GAC9IterationOyxq__GyYbq_YKYAcFZ
```

Comment:
Please change `test/abi/macOS/arm64/observation.swift` instead of editing the baselines. (you also don't need to edit the assert test unless there's an asserts only API)

Reply:
the other file is not discoverable btw... the error should indicate exactly what needs to happen instead - and also it should ideally not require mangled names too >.>